.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Lyrebird Java Client Release
+
+on: 
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Deploy to Github Package Registry
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        mkdir ~/.m2
+        echo "<settings><servers><server><id>github</id><username>Meituan-Dianping</username><password>${GITHUB_TOKEN}</password></server></servers></settings>" > ~/.m2/settings.xml
```

Comment:
echo ${GITHUB_TOKEN} will display your token to CI terminal?

Reply:
@zhaoye see 
036323e